### PR TITLE
[docs][cloud] Add go application

### PR DESCRIPTION
**Pull request by ddhodge**
Add simple Go application example

@netlify /latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/

### CODE REVIEW

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+```
+
+### createDatabase
+
+The `createDatabase` method uses PostgreSQL-compliant DDL commands to create a sample database.
+
+```go
+Statement stmt = conn.createStatement();
+
+stmt = `CREATE TABLE DemoAccount (
```
Comment (dmagda) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md:123]: Please use the following code snippet here:

```
stmt := `DROP TABLE IF EXISTS DemoAccount`
_, err := db.Exec(stmt)
checkIfError(err)

stmt = `CREATE TABLE DemoAccount (
                      id int PRIMARY KEY,
                      name varchar,
                      age int,
                      country varchar,
                      balance int)`

_, err = db.Exec(stmt)
checkIfError(err)

stmt = `INSERT INTO DemoAccount VALUES
              (1, 'Jessica', 28, 'USA', 10000),
              (2, 'John', 28, 'Canada', 9000)`

_, err = db.Exec(stmt)
checkIfError(err)
```

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+var name, country string
+var age, balance int
+
+for rows.Next() {
+    err = rows.Scan(&name, &age, &country, &balance)
+    checkIfError(err)
+
+    fmt.Printf("name = %s, age = %v, country = %s, balance = %v\n",
+        name, age, country, balance)
```
Comment (dmagda) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md:153]: You're missing `}` in the end - you need to close the `for rows.Next() {...}` block.

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+```
+
+### transferMoneyBetweenAccounts
+
+The `transferMoneyBetweenAccounts` method updates your data.
+
+```go
+_, err = tx.Exec(`UPDATE DemoAccount SET balance = balance - $1 WHERE name = 'Jessica'`, amount)
```
Comment (dmagda) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md:161]: Use this code snippet:

```
tx, err := db.Begin()
checkIfError(err)

_, err = tx.Exec(`UPDATE DemoAccount SET balance = balance - $1 WHERE name = 'Jessica'`, amount)
if checkIfTxAborted(err) {
return
}
_, err = tx.Exec(`UPDATE DemoAccount SET balance = balance + $1 WHERE name = 'John'`, amount)
if checkIfTxAborted(err) {
return
}

err = tx.Commit()
if checkIfTxAborted(err) {
return
}
```

**Review by dmagda — COMMENTED** [review]
Please consider minor suggestions related to the code snippets. Thanks!

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+Import the Go PostgreSQL driver.
+
+```sh
+$ go get github.com/lib/pq
+```
+
+To start the application, run the following command.
+
+```sh
+go run sample-app.go
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md:74]:
```suggestion
$ go run sample-app.go
```

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+$ export GO111MODULE=auto
+```
+
+Import the Go PostgreSQL driver.
+
+```sh
+$ go get github.com/lib/pq
+```
+
+To start the application, run the following command.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md:71]:
```suggestion
Start the application.
```

File: docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+
+```sh
+go run sample-app.go
+```
+
+You should then see output similar to the following:
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md:77]:
```suggestion
You should see output similar to the following:
```

**Review by polarweasel — APPROVED** [review]
Preemptively approving. Small comments.

**Pull request merged**